model/src/main/java/com/javahelp/model/userEntityTest/User.java

```diff
@@ -0,0 +1,54 @@
+package com.javahelp.model.userEntityTest;
+import com.javahelp.model.user.UserInfo;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.UserType;
+import com.javahelp.model.user.ProviderUserInfo;
+
+public class User {
+    /**
+     * A class for a user entity
+     */
+
+    private String stringID;
+    private UserInfo userInfo;
+
+    /**
+     * A constructor for the current User Entity
+     *
+     * @param stringID - the ID of the account
+     * @param userInfo - The account information.
+     */
+
```

Comment:
This whitespace shouldn't be here, Javadocs should be directly above the item they refer to.